content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -2,22 +2,22 @@
 This is an article about Sockets. It will give you a glimpse of how clients and servers communicate over the internet.
 
 Table of content:
-- [What is a socket](#what-is-a-socket)
-- [Programming on Server-side](#programming-on-server-side)
-- [](#)
-- [](#)
-- [](#)
+- [socket definition](#socket-definition)
+- [Server-Side Programming](#server-side-programming)
```

Comment:
I would move the table of contents to be after the introduction, and change:

> Table of content:

To: 

>### Table of contents:

For better formatting and grammar